eng/pipelines/libraries/stress/http.yml

```diff
@@ -51,23 +52,24 @@ jobs:
   displayName: Docker NanoServer
   timeoutInMinutes: 120
   pool:
-    vmImage: 'windows-latest'
+    name: NetCorePublic-Pool
+    queue: BuildPool.Windows.10.Amd64.VS2019.Open
```

Comment:
Build pool where docker was added is `BuildPool.Server.Amd64.VS2019`; should have same workloads as Windows.10 version

Reply:
@MattGal  It's [still failing](https://dev.azure.com/dnceng/public/_build/results?buildId=667588&view=logs&j=2d2b3007-3c5c-5840-9bb0-2b1ea49925f3&t=ac0b0b0f-051f-52f7-8fb3-a7e384b0dde9&l=12) due to a missing docker-compose. Could you please share any ETA for fixing this? It's getting urgent for us since I noticed an instability on Linux tests and we need to have it merged into master and tracked regularly.